dockerfile

```
@@ -0,0 +1,60 @@
+####################################################
+# To build your own image: 
+#    > docker build --build-arg BUILD_VERSION=0.0.3 --build-arg BUILD_DATE=2019-05-19 -t probot/smee-io:latest .
+#      (Protips: Add an another -t probot/smee-io:0.0.3 for a tag also for the version)
+# To run your image once it's ready:
+#    > docker run -d -p 3000:3000 --name smee-io probot/smee-io:latest
+# To push the image into your docker repository (e.g.):
+#    > docker push probot/smee-io:latest
+# If you wish to remove your dangling images after build, please do the following:
+#    > docker rmi $(docker images -f “dangling=true” -q)
+####################################################
+
+##############################################################################
+# Build arguments
+##############################################################################
+ARG BUILD_VERSION
+ARG BUILD_DATE
+ARG PORT=3000
+
+##############################################################################
+# Build the container with Source code compiled
+##############################################################################
+FROM node:10-alpine as build-env

```

Since we deploy smee.io to Heroku, it'd be nice to match their default Node version, which is LTS; so we should do:

```suggestion
FROM node:lts-alpine as build-env
```

---

done